### PR TITLE
Explore app metaverse api updates

### PR DESCRIPTION
**Pull request by Madders**
Blended the metaverse api places data in to the list with the beacon data. Also lightened the icons at the top as they were a little lost.

### CODE REVIEW

File: scripts/communityScripts/explore/explore.html [hunk 1]
```diff
@@ -286,22 +294,76 @@
...
+            if (locationData != null) {
+                for (var i = 0; i < locationData.length; i++) {
+                    joinedData.push({
+                        "index": joinedData.length
+                        , "location": locationData[i].Visit
+                        , "name": locationData[i]['Domain Name']
+                        , "owner": locationData[i].Owner
+                        , "userCount": locationData[i].People
+                        , "description": ""
+                        , "thumbnail": ""
```
Comment (ctrlaltdavid) [inline scripts/communityScripts/explore/explore.html:315]: Commas should be at end of lines, here and similarly below.

File: scripts/communityScripts/explore/explore.html [hunk 2]
```diff
@@ -286,22 +294,76 @@
...
-                "action": "requestAddressList",
-            };
-            EventBridge.emitWebEvent(JSON.stringify(readyEvent));
+            // if metaverse data not loaded fire a request for it
+            if (placesData == null) {
+                // TODO: fire off data get against the metaverse
+                axios.get("https://metaverse.vircadia.com/live/api/v1/places")
+                    .then((response) => {
+                        placesData = response.data.data.places;
+
```
Comment (ctrlaltdavid) [inline scripts/communityScripts/explore/explore.html:360]: Unnecessary blank line.

**Review by ctrlaltdavid — CHANGES_REQUESTED** [review]
Copy of Axios license needed, also.

File: scripts/communityScripts/explore/explore.html (hunk 2 above)
Comment (Madders) [reply scripts/communityScripts/explore/explore.html]: Updates have been committed to address those comments.

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (macOS-latest, client)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1315/Vircadia-Interface-PR1315-370e823-_1a02b918.dmg

**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1315/Vircadia-PR1315-370e823-_1a02b918.dmg

**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1315/Vircadia-PR1315-370e823-_1a02b918.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1315/Vircadia-PR1315-370e823-.tar_1a02b918.Z
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1315/Vircadia-PR1315-370e823-.tar_1a02b918.gz

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1315/Vircadia-PR1315-370e823-_1a02b918.exe

**Review by ctrlaltdavid — APPROVED** [review]

**Comment by AleziaKurdis** [comment]
Tested with success.
The number of places match: 
"Beacon entries" + "Active places from API where the the current protocol matches the returned by Interface"

The image (thumbnail) are displayed for the place that has one.
So I thing we are good to go with this PR.

-------
Doing this test allowed me to find a bug on the "metaverse server" or in the "domain server" (unclear which one is in cause):
https://github.com/vircadia/Iamus/issues/98

**Review by digisomni — APPROVED** [review]
I had tested this successfully. However it takes a while for the user count to update it would seem.

**Comment by AleziaKurdis** [comment]
> 
> 
> I had tested this successfully. However it takes a while for the user count to update it would seem.

Yes, as observed with other script using the Place API, this is the heartbeat frequency that does that.

**Pull request merged**